Sources/Error Handling/BackendError.swift

```diff
@@ -116,6 +117,13 @@ extension BackendError: PurchasesErrorConvertible {
                                                         fileName: source.file,
                                                         functionName: source.function,
                                                         line: source.line)
+        case let .invalidAppleSubscriptionKey(source):
+            return ErrorUtils.unknownError(
```

Comment:
Would it make sense to use `.configurationError` here?

Reply:
Definitely! Will switch it